vtr_flow/benchmarks/noc/Test_Designs/test_design_for_noc_traffic_flows_feature/multiple_noc_routers.flows

```diff
@@ -0,0 +1,7 @@
+<traffic_flows>
+
+	<single_flow src="^router:noc_router_four\|flit_out_two\[0\]~reg0$" dst="^router:noc_router_two\|flit_out_two\[0\]~reg0$" bandwidth="56" latency_cons="67" />
```

Comment:
Might want to use more realistic data; e.g. 100 MB/s and 30 ns

Reply:
Done